packages/calypso-e2e/src/media-helper.ts

```diff
@@ -26,12 +32,32 @@ export function getScreenshotDir(): string {
  * @returns {string} Absolute path to the directory.
  */
 export function getVideoDir(): string {
-	return path.resolve(
-		process.env.TEMP_ASSET_PATH || path.join( __dirname, '..' ),
-		process.env.VIDEODIR || 'screenshots/videos'
-	);
+	return path.resolve( getAssetDir(), process.env.VIDEODIR || 'screenshots/videos' );
```

Comment:
Do you think it would be useful to have the default env vars defined in one place? Maybe as a part of the `config`? So far I've noticed these: 
- `BROWSERLOCALE`
- `BROWSERSIZE`
- `SCREENSHOTDIR`
- `VIDEODIR`

Reply:
I considered your suggestion and think that putting it in the `config` file would be the best course of action as I prefer to not add dependencies if possible. 

The checks then could look like:
`process.env.VIDEODIR || config.get('videodir');`

We will still need to check the environment variables since at least one of the values used above, `SCREENSHOTDIR` changes depending on what magellan config is used.

I think with Sergio's proposal to migrate our test runner to Jest (#52786) we can get rid of magellan and that might be a good time to re-evaluate how various environment variables are stored/used.